lavalink/PlayerManager.py

```diff
@@ -232,16 +232,16 @@ def get(self, guild_id):
     async def clear_guild(self, guild_id):
         """ Removes the player and clears the cache. """
         try:
-            await self._players[guild_id].disconnect()
-            del self._players[guild_id]
+            player = self._players.pop(guild_id)
+            await player.disconnect()
         except KeyError:
             pass
 
-    def remove(self, guild_id):
+    async def remove(self, guild_id):

```

Comment:
Perhaps this method could also `del` the player and remove the need for `clear_guild` (maybe a boolean parameter to cleanup the player?)

Reply:
It's already removing the player with pop, and yes remove clear_guild and add param for it to call cleanup

Reply:
👌